Rakefile

```diff
@@ -29,3 +36,87 @@ RSpec::Core::RakeTask.new(:spec) do |r|
 end
 
 task :default => :spec
+
+
+class EmptyMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    if @app.respond_to?(:call)
+      status, headers, body = @app.call(env)
+    else
+      [200, {}, 'Hello world']
+    end
+  end
+end
+
+class FaradayMiddleware
+  def initialize(app)
+    @app = app
+  end
+
+  def call(env)
+    conn = Faraday.new(url: 'http://www.google.com') do |faraday|
+      faraday.use ZipkinTracer::FaradayHandler
+      # default Faraday stack
+      faraday.request :url_encoded
+      faraday.adapter Faraday.default_adapter
+    end
+   # conn.get('/')
+    [200, {}, 'hello world']
+  end
+end
+
+task :run_once do
+  empty_app = EmptyMiddleware.new(nil)
+  app = FaradayMiddleware.new(nil)
+
+  logger = Logger.new(Tempfile.new('fakelog'))
+  null_configuration = { logger: logger, sample_rate: 1}
+  null_tracer_rack = ZipkinTracer::RackHandler.new(app, null_configuration)
+  env = Rack::MockRequest.env_for('/path', {})
+
+  sleep(10)
+  null_tracer_rack.call(env)
+end
+
+# This task is used to help development of ZipkinTracer.
```

Comment:
maybe that comment could be removed or turned into something more useful, like explain what kind of benchs it's doing etc.